src/lib/containers/drug_upload_tool/DrugUploadForm.tsx

```diff
@@ -96,6 +96,7 @@ export default class DrugUploadForm extends React.Component<
   excludeWarningHeader = 'Skip This Step?'
   unsavedDataWarning = `You might have some unsaved data. Are you sure you want to leave?`
   formRef: any //ref to form for submission
+  formDivRef: any // ref to the div containing form (for scrolling on validation failure)
```

Comment:
can add a type to this in the future - `formDivRef: React.RefObject<HTMLDivElement>`